src/crimson/os/seastore/segment_manager/block.cc

```diff
@@ -291,6 +322,19 @@ Segment::write_ertr::future<> BlockSegment::write(
   return manager.segment_write({id, offset}, bl);
 }
 
+Segment::write_ertr::future<> BlockSegment::writev(
+  segment_off_t offset, ceph::bufferlist bl)
+{
+  if (offset < write_pointer || offset % manager.superblock.block_size != 0)
```

Comment:
i'd suggest add curly brackets around the statement even if it is just a single line, just for better readability.